### DISCUSSION TITLE
Is it possible to use a @nuxt/kit module in a nuxt bridge application ?

### DISCUSSION THREAD

**Discussion by Suniron** [nuxt/bridge#693] [Q&A]
Is it possible to use a @nuxt/kit module in a nuxt bridge application ? 

I got this error in the browser console:
```
[Vue warn]: Unknown custom element: <XButton> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```

The same module in a Nuxt 3 application works perfectly.

Typically, it's just this package without modification: https://github.com/gurvancampion/turborepo-nuxt-boilerplate/tree/main/packages/ui

**Comment by ColinRosati** [comment]
Yes you can!

The error message you got comes from missing the component name, so infact the module works. I imagine your trying to use `addComponent`

Check how the options are passed for name declaration. 
[addComponent opts](https://github.com/nuxt/nuxt/blob/1f38863c064b5700ef7a06a241dd0160aa788855/packages/kit/src/components.ts#L35)

Can you share how you are using `@nuxt/kit`.

Whats your vue version? theres a compatibility note there

**Reply by Suniron** [reply]
Sorry I will close this issue. The problem was I imported a component which import itself a Vue3 lib / nuxt 3 plugin
